webapi/_test_data/test_route/texts/beol/testLetter/knoraXmlImport.xsd

```diff
@@ -0,0 +1,198 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this a duplicate of `webapi/src/main/resources/knoraXmlImport.xsd`?

Reply:
I guess it is.

I downloaded the zip file and put everything in this directory so I can check the bulk xml file for validity.

Reply:
ok, I link to the original `knoraXmlImport.xsd`now https://github.com/dhlab-basel/Knora/pull/862/commits/879a8317ba775bd4632e81cc01a61c60abe9380e